crypto/siphash/siphash.c

```diff
@@ -75,5 +77,6 @@ uint64_t SIPHASH_24(const uint64_t key[2], const uint8_t *input,
   siphash_round(v);
   siphash_round(v);
 
-  return v[0] ^ v[1] ^ v[2] ^ v[3];
+  uint64_t b = v[0] ^ v[1] ^ v[2] ^ v[3];
+  return CRYPTO_load_u64_le(&b);
```

Comment:
This introduces a little more work for the existing LE path. How do you see the tradeoff of simplifying our code (same for both LE and BE) and performance? In other functions we've handled the two orders differently, why not here?

Reply:
My bad. This wasn't needed.